packages/tools/viewer-alpha/src/viewer.ts

```diff
@@ -368,12 +381,31 @@ export class Viewer implements IDisposable {
             });
 
             const camera = new ArcRotateCamera("Viewer Default Camera", 0, 0, 1, Vector3.Zero(), scene);
+            camera.useInputToRestoreState = false;
+
+            scene.onPointerObservable.add((pointerInfo) => {
+                if (pointerInfo.type === PointerEventTypes.POINTERDOUBLETAP) {

```

Comment:
Nit picky, but - the observables add function accepts a filter as a second variable, so if you have only one type you want to react to you can pass it there and the observer will be filtered at a lower level. Will prevent this from running needlessly during move, down, up, etc. (You can also pass two types with bitwise or)